site/source/tutorials/001_static_content/demo/finished/biz-e-corp/tests/unit/widgets/HelloWorld.ts

```diff
@@ -9,7 +9,8 @@ registerSuite({
 		const helloWorld = new HelloWorld();
 
 		const vnode = <VNode> helloWorld.__render__();
-		assert.strictEqual(vnode.vnodeSelector, 'div');
-		assert.equal(vnode.text, 'Hello, Dojo World!');
+		assert.strictEqual(vnode.vnodeSelector, 'h1');
+		assert.equal(vnode.text, 'Biz-E Bodies');
+		assert.equal(vnode.properties.title, 'I am a title!');
```

Comment:
The test runner throws a warning that `vnode.properties` can be undefined here. Changing this to `assert.equal(vnode.properties && vnode.properties.title, 'I am a title!');` will remove the warning.